src/com/aca/narekhakobyan/homework1/employee/GadgetManager.java

```diff
@@ -0,0 +1,182 @@
+package com.aca.narekhakobyan.homework1.employee;
+
+import com.aca.narekhakobyan.homework1.employee.functionality.Employee;
+import com.aca.narekhakobyan.homework1.employee.functionality.Management;
+import com.aca.narekhakobyan.homework1.items.concepts.*;
+import com.aca.narekhakobyan.homework1.items.models.*;
+import javafx.util.Pair;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class GadgetManager implements Management, Employee {
+
+    private ArrayList<Gadget> gadgets = new ArrayList<Gadget>();
+    private int soldAMD = 0;
+    private int stockAMD = 0;
+
+    private void calculateStockAMD() {
+        for (Gadget gadget : this.gadgets) {
+            this.stockAMD += gadget.getPrice() * gadget.getCount();
+        }
+    }
+
+    public GadgetManager(ArrayList<Gadget> gadgets) {
+       for (Gadget gadget : gadgets) {
+           this.addItem((Gadget)gadget);
+       }
+       calculateStockAMD();
+    }
+
+    @Override
+    public void listOfItems() {
+        for (Gadget gadget : this.gadgets) {
+            gadget.print();
+        }
+    }
+
+    public void listOfItemsVideoGadgets() {
+        for (Gadget gadget : gadgets) {
+            if (gadget instanceof GadgetVideoWithMemory) {
+                gadget.print();
+            }
+            if (gadget instanceof GadgetVideoWithoutMemory) {
+                gadget.print();
+            }
+            if (gadget instanceof GadgetRecordVideoWithMemory) {
+                gadget.print();
+            }
+        }
+    }
+
+    public void listOfItemsRecordGadgets() {
+        for (Gadget gadget : gadgets) {
```

Comment:
the same here, if you have print() method in Gadget why use instanceof checkings ?